libraries/AP_RTC/AP_RTC.cpp

```diff
@@ -83,6 +104,32 @@ void AP_RTC::set_utc_usec(uint64_t time_utc_usec, source_type type)
     // update signing timestamp
     GCS_MAVLINK::update_signing_timestamp(time_utc_usec);
 #endif
+
+#if DEBUG_RTC_SHIFT
+    uint64_t new_utc = 0;
+    UNUSED_RESULT(get_utc_usec(new_utc));
+    if (old_utc != new_utc) {
```

Comment:
this would seem to log every update? maybe log only for jumps of move than a threshold? or disable DEBUG_RTC_SHIFT by default?

Reply:
> this would seem to log every update? maybe log only for jumps of move than a threshold? or disable DEBUG_RTC_SHIFT by default?

Ah, crap.  Being a debug macro, it's *supposed* to be off by default!  My bad.